tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -896,6 +896,23 @@ public function theUsersOwnNameShouldNotBeListedInTheAutocompleteList() {
 		);
 	}
 
+	/**
+	 * @Then user :username should be listed in the autocomplete list on the webUI
+	 *
+	 * @param string $username
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userShouldBeListedInTheAutocompleteListOnTheWebui($username) {
+		$name = $this->sharingDialog->getAutocompleteItemsList();
```

Comment:
Sorry for being nitpicky here, but, I guess, this returns array here.
```suggestion
		$names = $this->sharingDialog->getAutocompleteItemsList();
```